event_session_registration_multi_qty/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_event_session_registration_multi_qty
```

Comment:
You can use a smaller filename instead of test_event_session_registration_multi_qty